### PR TITLE
Try to fix a bug and add lastfm ban API invoking

### PR DESCRIPTION
**Pull request by microdog**
Try to fix scrolling bug in main page on Windows.
Add invoking of lastfm ban API when do blockSong in player.

Hello,
The main page can be scrolled under Windows system, I tried to fix it and it works in my browser(Chrome Version 31.0.1650.4 m).
And I also add invoking of lastfm ban API when do blockSong in player.
I'm sorry that I did all these in the same branch, it's my fault.
Thanks

### CODE REVIEW

File: css/style.v2.css [hunk 1]
```diff
@@ -70,7 +70,7 @@ body {
     height: 40px;
     background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#f6f6f6), to(#efefef));
     background-image: -webkit-linear-gradient(top, #f6f6f6, #efefef);
-    position: absolute;
+    position: fixed;
```
Comment (fanzeyi) [inline css/style.v2.css:73]: Is this a bug? When will it happen? Could you give me some screenshot about this bug? thanks =D

File: coffee/player-lastfm.coffee [hunk 2]
```diff
@@ -44,6 +44,12 @@ define ["jquery"], ($) ->
                 authToken: md5(params.username + md5(params.password))
             , callback, "POST"
 
+        ban: (params, callback)->
+            @_api "track.ban",
```
Comment (fanzeyi) [inline coffee/player-lastfm.coffee:48]: What happened while calling this API? I've checked the [API Document](http://www.last.fm/api/show/track.ban) but I can't figure out what this api actually do. Is this a API for the option `Remove from your Library`?

Screenshot for the option `Remove from your Library`

screen shot 2013-11-08 at 3 27 59 am

File: css/style.v2.css (hunk 1 above)
Comment (microdog) [inline css/style.v2.css:73]: This is the normal state of the window.
normal
But if I scroll it using mouse wheel, the page will be scrolled and you can see a blank at the bottom.
scrolled

Issue exists in both Version 31 and Version 32.
Screenshot taken in: Windows 8.1, Chrome Version 31.0.1650.4 m.

But this issue does not exist in OS X 10.8 and 10.9, latest Chrome Dev Channel Version.

Thank you~ =w=

File: coffee/player-lastfm.coffee (hunk 2 above)
Comment (microdog) [inline coffee/player-lastfm.coffee:48]: Banning a track means that it won't play on your radio, ever again. It's an action for the radio service, used to teaching the site(or customizing your profile?) that you don't like this song.
The "Do not play again" function on douban.fm radio is similar to "Ban" on last.fm radio.
But this is a feature only meaningful to users who can still use the last.fm radio service.
Thanks~

**Comment by fanzeyi** [comment]
Merged.

Thank you! :beer:

**Comment by microdog** [comment]
:D
2013-11-8 ÏÂÎç4:32ÓÚ "Zeray Rice" notifications@github.comÐ´µÀ£º

**Pull request merged**